README.md

```diff
@@ -18,56 +21,60 @@
+When you have created an appropriate trello board you need to add the board id to the .env file. The board id can be found from the URL of the board, the url has the format of `https://trello.com/b/<boardID>/<boardName>`.
 
-You will also require a trello api key and api token. In order to generate these, ensure you are logged in to trello and then navigate to https://trello.com/app-key. At the top of the page will be your api key, and then below that a link to manually generate a Token. Clicking on this link will open a screen request you to confirm you are granting access to your boards, when you confirm this you will be presented with an api token. You should save the api key and token in your .env file, as the values of the TRELLO_API_KEY and TRELLO_API_SECRET variables respectively.
+You will also require a trello api key and api token. In order to generate these, ensure you are logged in to trello and then navigate to <https://trello.com/app-key>. At the top of the page will be your api key, and then below that a link to manually generate a Token. Clicking on this link will open a screen request you to confirm you are granting access to your boards, when you confirm this you will be presented with an api token. You should save the api key and token in your .env file, as the values of the TRELLO_API_KEY and TRELLO_API_SECRET variables respectively.
 
 More information about generation api keys and tokens for trello can be found [here](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/)
 
-
 ## Running the app using docker
 
 There is a multi-stage docker file for this project, containing a production build and a development build:
 
-### Production:
+### Production
+
 - To produce the image: `docker build --target production --tag todo-app:prod .`
 - To run: `docker run -p 5100:80 -d --env-file .env todo-app:prod`
 
 The app will then run be accessible on localhost:5100.
 
-### Development: 
+### Development
 
-The development container can be launched using `docker-compose up -d`. The app can then be accessed at localhost:5000 and will reload whenever changes are made to the app files locally.
+The development container can be launched using `docker-compose up -d --build`. The app can then be accessed at localhost:5000 and will reload whenever changes are made to the app files locally.
 
  If you would prefer to build and run the image and container manually:
+
 - To produce the image: `docker build --target development --tag todo-app:dev .`
 - To run `docker run -p 5000:5000 -d --env-file .env --mount type=bind,source="$(pwd)",target=/app todo-app:dev`
-  
 
+### Test
 
+- To produce the image: `docker build --target test --tag todo-app:test .`
+- To run `docker run  --env-file .env --mount type=bind,source="$(pwd)",target=/app todo-app:test`
```

Comment:
You don't actually need  `--env-file .env` here or for the run todo-app:dev commands because it's included in the bind mount.